### PR TITLE
Add Naxe machines, quality and materials

### PR DESCRIPTION
**Pull request by Naxecorp**
Hello there, 

Here is some files for adding Naxe manufacturer in the Cura software.
We are based in France. 

Thank you all guys

### CODE REVIEW

File: resources/materials/naxe_ABS.xml.fdm_material [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0' encoding='utf-8'?>
```
Comment (nallath) [inline resources/materials/naxe_ABS.xml.fdm_material:1]: Materials need to go in https://github.com/ultimaker/fdm_materials

File: resources/materials/naxe_ABS.xml.fdm_material [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura" version="1.3">
+  <metadata>
+    <name>
+      <brand>Naxe</brand>
```
Comment (nallath) [inline resources/materials/naxe_ABS.xml.fdm_material:5]: Is this really a fundamentally different material? It seems like it's better to just add machine settings to generic 1.75 ABS instead

**Review by nallath — CHANGES_REQUESTED** [review]
The STL is also too big. We require buildplate meshes to be well below 1mb.

**Comment by Naxecorp** [comment]
Thank you!
please have a look to my new commit ;)

**Comment by nallath** [comment]
Apologies! I've noticed another mistake while i was testing this. Because you set the material_print_temperature in the quality files to have a certain value (eg; a hardcoded one), it's no longer using whatever is defined in the material. 

So as a rule of thumb, it's better to use 
material_print_temperature = =default_material_print_temperature + 10 
If you want to use the default value of the material but print it at a higher temp.

**Comment by Naxecorp** [comment]
Hi nallath, 
Thanks for reply, I have made the changes. Feel free to comment this out.

**Pull request merged**